src/Parsers/Tree/Statements/DropNode.h

```diff
@@ -0,0 +1,31 @@
+#ifndef __DROP_NODE_H_
+#define __DROP_NODE_H_
+
+#include <string>
+
+#include "StatementNode.h"
+
+namespace FQL
+{
+    class DropNode : public StatementNode
+    {
+    protected:
+        // TODO: change to custom type.
+        const std::string dir;
+
+        const bool dropIfExists = false;
+
+    public:
+        DropNode(const char *dir, bool dropIfExists = false);
```

Comment:
Let's omit this constructor to be less dependent on `C` to avoid errors. 😀

There are other cases I think.

Reply:
Yeah, I just added it to follow ur convention but I did not like it.